metrics/web/lib/dashboard/presentation/widgets/strategy/metric_value_theme_strategy.dart

```diff
@@ -11,7 +11,8 @@ import 'package:metrics/common/presentation/metrics_theme/widgets/strategy/value
 /// * if the percent value is greater than [lowPercentBound] and less than [mediumPercentBound], applies the [MetricCirclePercentageThemeData.lowPercentTheme]
 /// * if the percent value is greater than or equal to [mediumPercentBound] and less than [highPercentBound], applies the [MetricCirclePercentageThemeData.mediumPercentTheme]
 /// * if the percent value is greater or equal to [highPercentBound], applies the [MetricCirclePercentageThemeData.highPercentTheme].
-class MetricValueThemeStrategy implements ValueBasedThemeStrategy<double> {
+class MetricValueThemeStrategy
```

Comment:
```suggestion
class MetricsValueThemeStrategy
```
to make it consistent.